bin/console

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+require "bundler/setup"
+require "solid_assert"
+
+require "irb"
+IRB.start
```

Comment:
I know this PR is already merged, but as long as you are cleaning/updating things, you might as well get rid of this script. The same functionality is already available via `bundle console` (no need to create your own `bin/console` anymore).